woocommerce/woocommerce-ios
# Remove Woo Express sign up from prologue screen

Closes: #12324 


## Description

This PR updates the prologue screen to replace the store creation CTA on the prologue screen with a link button to show hosting options for site creation.

Please also review https://github.com/wordpress-mobile/WordPressAuthenticator-iOS/pull/844.

## Testing instructions

- Enable the feature flag `.noMoreWooExpressSignup` and build the app.
- Log out of the app if needed.
- Confirm that the Log In button has primary style.
- Confirm that the prologue screen no longer displays the CTA to store creation. Instead, a link button "Starting a new store?" is displayed below the Log In button.
- Tap "Starting a new store?", confirm that a web view is displayed showing the content of https://woo.com/hosting-solutions/.
- Disable the feature flag `.noMoreWooExpressSignup`, confirm that the prologue screen looks like before.

## Screenshots


https://github.com/woocommerce/woocommerce-ios/assets/5533851/dfb2b3bc-3501-42f8-ae4d-6307c6a8ce31



---
